website/source/docs/configuration/seal/gcpckms.html.md

```diff
@@ -35,6 +35,21 @@ seal "gcpckms" {
 }
 ```
 
+## Setting the Google Compute Instance Scope
+If you wish to use auto-unseal without having a credentials file on the instance, you can set the cloudkms scope on your instance.  Here is an example command:
```

Comment:
The new website is more strict about formatting of markdown and I believe it requires an empty line between a heading and text.